BasicPlybot.yaml

```diff
@@ -460,10 +545,10 @@ Properties:
       Number, validators: null, value: 2619}
   Export Subsystems: !!robotbuilder.data.properties.BooleanProperty {default: true,
     name: Export Subsystems, validators: null, value: true}
-  Wiring file location: !!robotbuilder.data.properties.FileProperty {default: 'C:\Users\evangebo\',
+  Wiring file location: !!robotbuilder.data.properties.FileProperty {default: 'C:\Users\super\',
     extension: null, folder: true, name: Wiring file location, validators: null, value: 'C:\Users\evangebo\'}
-  Export Directory: !!robotbuilder.data.properties.FileProperty {default: 'C:\Users\evangebo\FRCProjects',
-    extension: null, folder: true, name: Export Directory, validators: null, value: 'C:\workspace'}
+  Export Directory: !!robotbuilder.data.properties.FileProperty {default: 'C:\Users\super\FRCProjects',
+    extension: null, folder: true, name: Export Directory, validators: null, value: 'D:\SourceTreeRepositories'}
   Autonomous Command: !!robotbuilder.data.properties.TypeSelectionProperty {default: None,
```

Comment:
@EvanGebo When you go to review PRs before merging, you'll want to make sure to check RobotBuilder's export directory. Most students will use C:\workspace. Power users, and those on macs will have something different. 
The quick fix for you would be right before merging, check the branch out, fix it, and commit (since this is practice, no need to).

Reply:
Does the default matter, or can that just be whatever?

Reply:
Default can be whatever. Main thing is the actual set value, which is what'll show up (and what it uses) when you open the file with RobotBuilder